src/js/angular/clustermanagement/controllers.js

```diff
@@ -189,24 +190,24 @@ function ClusterManagementCtrl($scope, $http, $q, toastr, $repositories, $modal,
             const loaderMessage = $translate.instant('cluster_management.delete_cluster_dialog.loader_message');
             $scope.setLoader(true, loaderMessage);
             ClusterRestService.deleteCluster(forceDelete)
-                .success((data) => {
-                    const allNodesDeleted = Object.values(data).every((node) => node === 'DELETED');
+                .then((response) => {
+                    const allNodesDeleted = Object.values(response.data).every((node) => node === 'DELETED');
```

Comment:
Is it `node` or it's a node status? Also didn't you create some enums for these statuses?